test/airport/_mocks/airportJsonMock.js

```diff
@@ -1,532 +1,532 @@
 /* eslint-disable */
 export const AIRPORT_JSON_KLAS_MOCK = {
-  "radio": {
```

Comment:
totally cool.  auto-indent away!  I just wanted to make sure it wasn't a legitimate change in the _data_.  that had me worried for a minute.